priv/repo/migrations/20200220012135_add_cap_to_rooms.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Ret.Repo.Migrations.AddCapToRooms do
+  use Ecto.Migration
+
+  def change do
+    alter table("hubs") do
+      add(:member_cap, :integer, null: false, default: 0)
```

Comment:
we should run an `UPDATE` to set the member_cap properly on existing rows to 64.

Reply:
you can either split this migration into a `up/down` and do the alter manually, or leave this one and have a separate migration with an `up/down` that just runs the update in the `up`

Reply:
Hmm, there are no existing room counts. My intent was that 0 means "server default", but maybe that is too weird.

Reply:
oh, sorry, duh. i think it should be nullable if we want to have semantics around a default